<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeshWindow</class>
 <widget class="QMainWindow" name="MeshWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MeshView -- zdd</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuRenderer">
    <property name="title">
     <string>Renderer</string>
    </property>
    <addaction name="actionDefault"/>
    <addaction name="actionPoints"/>
    <addaction name="actionWireframe"/>
    <addaction name="actionWireframe_light"/>
    <addaction name="actionWireframe_color"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionReset_View"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRenderer"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionDefault">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="actionPoints">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Points</string>
   </property>
  </action>
  <action name="actionWireframe">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Wireframe</string>
   </property>
  </action>
  <action name="actionReset_View">
   <property name="text">
    <string>Reset View</string>
   </property>
  </action>
  <action name="actionWireframe_color">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Wireframe (color)</string>
   </property>
  </action>
  <action name="actionWireframe_light">
   <property name="text">
    <string>Wireframe (light)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
